templates/kubernetes/terraform/modules/kubernetes/mysql/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "db_params" {
```

Comment:
A module should accept individual variables rather than a map, so it can define a clean, well-documented interface.